v2/rest/pulse.go

```diff
@@ -0,0 +1,136 @@
+package rest
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"path"
+	"strconv"
+
+	"github.com/bitfinexcom/bitfinex-api-go/pkg/convert"
+	"github.com/bitfinexcom/bitfinex-api-go/pkg/models/pulse"
+	"github.com/bitfinexcom/bitfinex-api-go/pkg/models/pulseprofile"
+	"github.com/bitfinexcom/bitfinex-api-go/v2"
+)
+
+type PulseService struct {
+	requestFactory
+	Synchronous
+}
+
+// PublicPulseProfile returns details for a specific Pulse profile
+// https://docs.bitfinex.com/reference#rest-public-pulse-profile
+func (ps *PulseService) PublicPulseProfile(nickname string) (*pulseprofile.PulseProfile, error) {
+	if (len(nickname)) == 0 {
+		return nil, fmt.Errorf("nickname is required argument")
+	}
+
+	req := NewRequestWithMethod(path.Join("pulse", "profile", nickname), "GET")
+	raw, err := ps.Request(req)
+	if err != nil {
+		return nil, err
+	}
+
+	pp, err := pulseprofile.NewFromRaw(raw)
+	if err != nil {
+		return nil, err
+	}
+
+	return pp, nil
+}
+
+// PublicPulseHistory returns latest pulse messages. You can specify
+// an end timestamp to view older messages.
+// see https://docs.bitfinex.com/reference#rest-public-pulse-hist
+func (ps *PulseService) PublicPulseHistory(limit, end int) ([]*pulse.Pulse, error) {
```

Comment:
I think we should use the `bitfinex.Mts` type here, just to keep it consistent with candle and trades hist.

For a future refactor (will be a breaking change) we could update all of these to use the built in golang time type.